HLTrigger/Configuration/test/OnLine_HLT_PIon.py

```diff
@@ -52,7 +52,7 @@
     test = cms.vstring( 'ECAL' ),
     emulator = cms.vstring( 'None' )
   ),
-  streamEventDisplay = cms.PSet( 
+ streamEventDisplay = cms.PSet( 
```

Comment:
Picky, but please undo this change too (as mentioned, these files are only updated by HLT).

Reply:
ok done